### DISCUSSION TITLE
【求助】喵吉长官好像遇到了麻烦！去看看吧........

### DISCUSSION THREAD

**Discussion by MiaoJiSir** [share121/inter-knot#836] [其他]
![微信图片_20240810093722](https://github.com/user-attachments/assets/75b23058-9b29-48ca-978e-d5e931dfea81)
帮助喵吉长官请到假期！绳匠们完成任务后，请到喵吉长官处领取奖励！

**Comment by ZhuYuan-148** [comment]
喵吉上班时间刷绳网，不做好吉祥物的本质工作，跟我回治安局
![Screenshot_20240810_173930](https://github.com/user-attachments/assets/37009e93-9525-4164-801d-f8e525cc3de6)

**Comment by XiaoFffeng** [comment]
我的天，朱鸢警官这么高强度冲浪，不会怕被发现吗

**Reply by ZhuYuan-148** [reply]
怎么会？这是工作的一部分，维护绳网秩序也是治安官的责任
